03-Mains/Heaving_Water_mass_transformation_decomposition.ipynb

```diff
@@ -0,0 +1,1863 @@
+{
```

Comment:
<p>The new function name is much clearer, thanks!</p><p> </p><p>I should have mentioned this in the first review, but it is generally better to include required variables in the function call, rather than relying on them as global variables. In this case, the function needs <code>start_time</code>, <code>end_time</code>, <code>lat_slice</code>, and <code>lon</code>, but none of them are included in the function call.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/620/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>